src/test/java/com/adobe/epubcheck/api/Epub30CheckExpandedTest.java

```diff
@@ -133,19 +133,22 @@ public void testValidateEPUBPLoremForeign()
   @Test
   public void testValidateEPUBPLoremObjectFallbacks()
   {
+    Collections.addAll(expectedWarnings, MessageId.RSC_017);
```

Comment:
Ideally for all these tests where the content is expected to be valid, the data files should be fixed instead of adding an expected warning.
Otherwise a long list of warnings and errors is gradually added to each an every test, and it gets harder to know what the test was actually suppose to focus on.

Reply:
Right, but there were a number of "valid" files that depended on deprecated features. That's typically where I added warnings (e.g., I believe the one referenced here was using bindings on the object's media type).